yew-router/examples/static/index.html

```diff
@@ -4,7 +4,7 @@
         <meta charset="utf-8">
         <title>Yew example</title>
         <script type="module">
-			import init from "./wasm.js"
+			import init from "/wasm.js"
```

Comment:
why was this changed?

Reply:
As mentioned in the issue description, this allows full-site refreshing an example when it's currently at an nested route. Otherwise it'll try to include include `/a/wasm.js` when currently being on `/a`, which doesn't work.

Reply:
Got it! Sorry I missed that. Sounds good!

Reply:
No worries. It wasn't in the description when first opening the issue, so that might be why.